source/wave_energy.rst

```diff
@@ -80,11 +78,11 @@ The ability of a WEC device to harvest wave energy can be expressed by wave ener
 
 .. Table 3.2 shows an example of wave energy absorption performances in each seastate bin for Pelamis.
  
-.. figure:: ./wave_energy/table_energyabsorption.png
-   :align: center
-   :figwidth: 500px
+.. csv-table:: **Wave energy absorption performance (kW) for Pelamis**
```

Comment:
A few things here:
- I noticed the `Figure` number changed. I also noticed a reference to `Table 3.1` that might have been that commented out `image002`? 
- I wonder if keeping this `png` table makes sense here, but later on for the **Data Needs**, it makes sense to show what that table really looks like as a data input?

Reply:
The `Table 3.1` reference seems to be about the seastate table. I just change it to say "The table below". And I'll put the png back here.